docs/devices/8850100.md

```diff
@@ -0,0 +1,25 @@
+<!-- !!!! -->
```

Comment:
As I mentioned in https://github.com/Koenkk/zigbee-herdsman-converters/pull/3933/files#r815450403, Not sure what the naming convention should be here given my verified test with one model but assumed compatibility with other models. Happy to make whatever changes are necessary.